### ISSUE TITLE
New federation level: Forward Reports

### ISSUE THREAD

**Issue by nesges** [mastodon/mastodon#24983]
### Pitch

I propose a federation level next to blocks and silence, which ensures that reports about users of the server are not submitted to the local moderation, but exclusively to the remote moderation.

### Motivation

Often the local moderation has its hands full with handling reports about users from a remote server. Although the moderation of the remote server does an excellent job and handles these reports as well (often resulting in user blocks and thus solving them), the local moderation has to handle the messages, which would not be necessary.

**Comment by ThisIsMissEm** [comment]
Does filtering the reports by account origin help you here?
 
![](https://user-images.githubusercontent.com/3387286/208268158-9ae8bcbe-9365-490c-bfac-b7cf5d31fdc8.png)

That way you can prioritise your work to the content local to your instance first, before looking at remote.

**Comment by ThisIsMissEm** [comment]
(Perhaps there'd be room for adding a "remember which origin you're looking at" feature, such that going to Reports would always take you to that filter view.

**Comment by nesges** [comment]
> Does filtering the reports by account origin help you here?

No, that wouldn't change the workload of handling those reports

**Comment by ThisIsMissEm** [comment]
@nesges right, however, it would at least allow you to choose what to address first. Not all forwarded reports will be acted on by the remote instance, so it is important that you still see them and make a decision about what to do. Forwarded reports can be a great source of information when it comes to figuring out whether or not you may need to enforce a domain block or silence for a specific instance.

**Comment by ThisIsMissEm** [comment]
@vmstan I'd recommend a won't fix for this issue, given that the local copy of a report about remote content is very necessary and an important part of moderating an instance and protecting your users from harm.

The only case where it may be beneficial to turn off is for single-user mode servers, where local moderation is a very different story to that of moderating communities.
